Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:xenial
+
+RUN \
+  apt-get update && \
+  apt-get -y install \
+    software-properties-common \
+    python3-pip \
+    git
+
+RUN \
+  pip3 install \
+    docutils==0.15 \
+    sphinx==1.6.2 \
+    python-docx==0.8.6 \
+    sphinx-bootstrap-theme==0.6.4 \
+    sphinxcontrib-websupport==1.0.1 \
+    git+https://github.com/mherkazandjian/docxsphinx.git@master
+
+RUN apt-get clean
+
+ENTRYPOINT ["make", "docx", "html"]
```

Comment:
How should one use the docker image? This is what I tried:
```
$ docker build -t docxsphinx .
$ docker run --mount . docxsphinx
make: *** No rule to make target 'docx'.  Stop.
```
So I guess it should be ran 'in' one of the example directories somehow, but I don't know how.